networkit/components.pyx

```diff
@@ -45,78 +45,99 @@ cdef class ConnectedComponents(Algorithm):
 		self._this = new _ConnectedComponents(G._this)
 
 	def getPartition(self):
-		""" Get a Partition that represents the components.
+		""" 
+		getPartition()
 
-		Returns:
-		--------
+		Get a Partition that represents the components.
+
+		Returns
+		-------
 		networkit.Partition
 			A partition representing the found components.
 		"""
 		return Partition().setThis((<_ConnectedComponents*>(self._this)).getPartition())
 
 	def numberOfComponents(self):
-		""" Get the number of connected components.
+		""" 
+		numberOfComponents()
+		
+		Get the number of connected components.
 
-		Returns:
-		--------
-		count:
+		Returns
+		-------
+		int
 			The number of connected components.
 		"""
 		return (<_ConnectedComponents*>(self._this)).numberOfComponents()
 
 	def componentOfNode(self, v):
-		"""  Get the the component in which node `v` is situated.
+		"""  
+		componentOfNode(v)
+		
+		Get the the component in which node `v` is situated.
 
+		Parameters
+		----------
 		v : node
 			The node whose component is asked for.
+
+		Returns
+		-------
+		int
+			Component in which node `v` is situated
 		"""
 		return (<_ConnectedComponents*>(self._this)).componentOfNode(v)
 
 	def getComponentSizes(self):
-		""" Get the component sizes.
+		""" 
+		getComponentSizes()
+		
+		Get the component sizes.
 
-		Returns:
-		--------
-		map:
-			The map from component to size.
+		Returns
+		-------
+		dict
+			A dict containing the component and their sizes.
```

Comment:
Maybe it is clearer something like "A dict of 'int: int', mapping component ids to their size."

Reply:
Yeah, we are also not really consistent concerning listing types for lists/dicts. I would maybe prefer something like:
```
dict(int : int)
    A dict, mapping component ids to their size.
```
I am not sure the `,` belongs there btw.

Reply:
Yes, this is even better, I think. I would not write a comma there.